libs/cli/langchain_cli/integration_template/integration_template/llms.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+from functools import partial
+from typing import (
+    Any,
+    AsyncIterator,
+    Iterator,
+    List,
+    Optional,
+)
+
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain_core.language_models import BaseLLM
+from langchain_core.outputs import GenerationChunk, LLMResult
+
+
+class Integration(BaseLLM):
```

Comment:
Can we use the `<Name>LLM` name convention? If so, it would be `IntegrationLLM`

Reply:
Updating! This would be moving forward, and we'll probably tolerate people naming a "default" thing. e.g. the `Chroma` or `FAISS` vector stores, or the LLMs you referenced.

Reply:
Thanks!!!!